graylog2-server/src/test/java/org/graylog2/utilities/date/NaturalDateParserTest.java

```diff
@@ -55,15 +54,54 @@ public void testParseFailsOnEmptyDate() throws Exception {
     }
 
     @Test
-    @Ignore
-    public void testTemporalOrder() throws Exception {
+    public void testDefaultTZ() throws Exception {
         NaturalDateParser p = new NaturalDateParser();
 
+        NaturalDateParser.Result today = p.parse("today");
+        assertThat(today.getFrom()).as("From should not be null").isNotNull();
+        assertThat(today.getTo()).as("To should not be null").isNotNull();
+        assertThat(today.getDateTimeZone().getID()).as("should have the Etc/UTC as Timezone").isEqualTo("Etc/UTC");
+    }
+
+    @Test
+    public void testUTCTZ() throws Exception {
+        NaturalDateParser p = new NaturalDateParser("Etc/UTC");
+
+        NaturalDateParser.Result today = p.parse("today");
+        assertThat(today.getFrom()).as("From should not be null").isNotNull();
+        assertThat(today.getTo()).as("To should not be null").isNotNull();
+        assertThat(today.getDateTimeZone().getID()).as("should have the Etc/UTC as Timezone").isEqualTo("Etc/UTC");
+    }
+
+    @Test
+    public void testEuropeBerlinTZ() throws Exception {
+        NaturalDateParser p = new NaturalDateParser("Europe/Berlin");
+
+        NaturalDateParser.Result today = p.parse("today");
+        assertThat(today.getFrom()).as("From should not be null").isNotNull();
+        assertThat(today.getTo()).as("To should not be null").isNotNull();
+        assertThat(today.getDateTimeZone().getID()).as("should have the Europe/Berlin as Timezone").isEqualTo("Europe/Berlin");
+    }
+
+    @Test
+    public void testInvalidTZ() throws Exception {
+        NaturalDateParser p = new NaturalDateParser("LOLWut");
+
+        NaturalDateParser.Result today = p.parse("today");
+        assertThat(today.getFrom()).as("From should not be null").isNotNull();
+        assertThat(today.getTo()).as("To should not be null").isNotNull();
+        assertThat(today.getDateTimeZone().getID()).as("should have the Etc/UTC as Timezone").isEqualTo("UTC");
```

Comment:
I think we should throw an exception if the tz id is invalid, instead of silently falling back to `UTC`.